### ISSUE TITLE
Incorrect total downloaded maps size is displayed in Download dialog

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#5019]
Reported from iOS (the total size is +50Mb more than the size of downloaded countries), need to check for Android too (but see below).

The issue is likely caused by counting the size of the World.mwm and WorldCoasts.mwm that are already included in the app bundle. For all builds where world files are embedded in the app's resources (excluding FDroid), their size should not be displayed in the Download dialog total map size summary.

**Comment by lapaz17** [comment]
@biodranik could you assign me?

**Comment by biodranik** [comment]
No. We don't assign first-time contributors. Just make a PR with the fix.
